static/panes.js

```diff
@@ -117,6 +117,12 @@ function Pane(id) {
   cloneButton.innerHTML = 'o';
   cloneButton.title = 'disconnect';
 
+  var saveButton = document.createElement('a');
+  saveButton.innerHTML = '&#10515;';
+  saveButton.title = 'save';
+  saveButton.download = 'untitled_image.jpg';
+  saveButton.href = '';
```

Comment:
Oh, I wasn't aware that the content can be unsavable. When can this happen? I thought the panes only show up for images and charts.